Add the USB ID for PFU fi-8170 scanner.  This
can be dropped once upstream sane-backends has
proper support.

--- a/backend/fujitsu.conf.in
+++ b/backend/fujitsu.conf.in
@@ -265,5 +265,9 @@
 #fi-7800
 usb 0x04c5 0x160b
 
+#fi-8170
+usb 0x04c5 0x15ff
+
+
 #ScanSnap iX1600
 usb 0x04c5 0x1632
--- a/doc/descriptions/fujitsu.desc
+++ b/doc/descriptions/fujitsu.desc
@@ -674,6 +674,11 @@
 :status :good
 :usbid "0x04c5" "0x160b"
 
+:model "fi-8170"
+:interface "USB"
+:status :good
+:usbid "0x04c5" "0x15ff"
+
 :model "ScanSnap iX1600"
 :interface "USB WiFi"
 :status :good
